ci/Jenkinsfile.gh-pages

```diff
@@ -14,29 +14,35 @@ pipeline {
     GIT_AUTHOR_NAME = 'status-im-auto'
     GIT_AUTHOR_EMAIL = 'auto@status.im'
     PUPPETEER_SKIP_DOWNLOAD = 'true'
+    NO_COLOR = 'true'
   }
 
   stages {
     stage('Deps') {
       steps {
-        sh 'npm ci'
+        sh 'npm install'
       }
     }
 
-    stage('Build') {
+    stage('Packages') {
       steps {
         sh 'npm run build'
+      }
+    }
+
+    stage('Build') {
+      steps {
         sh 'npm run doc'
       }
     }
 
     stage('Publish') {
       when { expression { GIT_BRANCH.endsWith('master') } }
-      steps { script {
+      steps { dir('packages/core') { script {
```

Comment:
If I understand correctly, for this step you are changing the current dir to `packages/core` but yet, you moved the `deploy` script to the root `package.json` folder. Not sure I understand how it's working.

Also, going to https://ci.infra.status.im/job/website/job/js.waku.org/ there is no build history, is that expected?

Reply:
Ah, you're right, we don't need this. It works purely because `npm` also reads `package.json` files in parent directories, which makes no fucking sense, but it is what it is.